generators/app/templates/public/locales/en-US/translation.json

```diff
@@ -1,4 +1,11 @@
 {
+  "appPermission": {
+    "message": "The application needs to have all the permissions to work.",
```

Comment:
Maybe this should read something like "The application permissions are not properly set. Please add additional permissions and try again".

Reply:
updated